service/src/main/kotlin/fi/espoo/evaka/dailyservicetimes/DailyServiceTimes.kt

```diff
@@ -33,21 +33,27 @@ enum class DailyServiceTimesType {
 
 @JsonTypeInfo(use = JsonTypeInfo.Id.DEDUCTION, property = "type")
 sealed class DailyServiceTimes(
+    open val childId: UUID,
```

Comment:
I don't understand the motivation behind adding `childId` to this class. Seems like unnecessary coupling.

Reply:
It was needed so that when querying service times for week view we know which row is for which child

Reply:
removed